sdk/appconfiguration/azure-data-appconfiguration-v2/pom.xml

```diff
@@ -56,14 +56,14 @@ Code generated by Microsoft (R) TypeSpec Code Generator.
     <dependency>
       <groupId>com.azure.v2</groupId>
       <artifactId>azure-core</artifactId>
-      <version>2.0.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure.v2:azure-core;dependency} -->
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.v2:azure-core;dependency} -->
     </dependency>
 
     <!-- Test scope -->
     <dependency>
       <groupId>com.azure.v2</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>2.0.0-beta.1</version> <!-- {x-version-update;unreleased_com.azure.v2:azure-identity;dependency} -->
+      <version>2.0.0-beta.1</version> <!-- {x-version-update;com.azure.v2:azure-identity;dependency} -->
```

Comment:
Not sure why the change here..

Reply:
I could only infer from the title that this PR could have something to do with the change:
https://github.com/Azure/azure-sdk-for-java/pull/45427

@alzimmermsft Would you help take a look?

Reply:
I guess for now we had to  manually revert these? (if not many)

Reply:
> but this seems need to be done for every increment version PR...

Yeah... Another one: https://github.com/Azure/azure-sdk-for-java/pull/45471/files

Seems we have to do manual revert for now.

Reply:
> I could only infer from the title that this PR could have something to do with the change: #45427
> 
> @alzimmermsft Would you help take a look?

Yeah, this is definitely something due to that PR. Will take a look into what's happening.